tokio-sync/src/mpsc/list.rs

```diff
@@ -215,8 +215,6 @@ impl<T> Tx<T> {
 
 impl<T> fmt::Debug for Tx<T> {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        use std::sync::atomic::Ordering::Relaxed;
```

Comment:
I'd rather keep this in a separate PR. I submitted https://github.com/tokio-rs/tokio/pull/1043.

Reply:
Ok, will wait for that to merge first then. It just caused my tests to fail.

Reply:
This has been updated already in the PR :+1: